### PR TITLE
Jhony/data api filtering

### PR DESCRIPTION
**Pull request by jfavellar90**

### CODE REVIEW

**Comment by felipemontoya** [comment]
@jfavellar90 are we ready to merge this? or have you changed something?

**Comment by felipemontoya** [comment]
I would like to squash all this commits

**Comment by jfavellar90** [comment]
Done the squash

File: common/djangoapps/edunext/viewsets.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+from paginators import DataApiResultsSetPagination
+
+
+class DataApiViewSet(mixins.ListModelMixin,
+                     viewsets.GenericViewSet):
+    """
+    A generic viewset for all the instances of the data-api
+    """
+    authentication_classes = (MicrositeManagerAuthentication,)
+    # permission_classes = (AccountPermission,)  # We need to authorize the request based on the microsite?
```
Comment (felipemontoya) [inline common/djangoapps/edunext/viewsets.py]: What about the Permission model?

**Comment by felipemontoya** [comment]
:+1:

**Pull request merged**